### PR TITLE
Improvement/reorganize folder

### PR DESCRIPTION
**Pull request by edd34**
The idea is to improve the folder structure of the project in order to make it more maintainable and upgradable.

I have done the following tasks :
1) Split EthLeverager.sol contract to :
* contract/CallerContract.sol
* contract/EthLeverager.sol
* contract/HelperContract.sol
* contract/Interfaces.sol

2) Split EthLeverageWallet.py to :
* src/EthLeverageWallet.py
* src/Helper.py

3) Move contracts/abi.txt and contracts/bytecode.txt to respectively build/abi.txt and build/bytecode.txt

4) Format the smartcontracts and python code

5) Update EthLeverageWallet.py to find abi and bytecode in build folder

6) change pragma solidity to allow the program to be compiled by solc v0.8.x using '^'

[x] You can still compile the project using `solc contract/EthLeverager.sol`
[x] I have tested the project using test.py and my own blockchain provider url, and it works well

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+from src.EthLeverageWallet import EthLeverage
 from web3 import Web3
 
-http_rpc = None # https://mainnet.infura.io/v3/...
+http_rpc = None  # https://mainnet.infura.io/v3/...
 
 # Automatically takes `w3.eth.accounts[0]` as the main account
-myContract = MyContract().init(http_rpc)
-myContract.buildContract()
+i_EthLW = EthLeverage().init(http_rpc)
```
Comment (edd34) [inline test.py:7]: I used the name EthLeverage  because it's more meaningful than MyContract. i_EthLW stands for "instance of EthLeverageWallet"

**Comment by edd34** [comment]
I have also updated the notebook with recent changes on folder structure

**Review by nerolation — CHANGES_REQUESTED** [review]
there is a type in the description (move EthLevergaWallet... instead of move EthLeverageWallet)

**Comment by edd34** [comment]
> there is a type in the description (move EthLevergaWallet... instead of move EthLeverageWallet)

Resolved

**Comment by nerolation** [comment]
approved

**Pull request merged**